andoriyaprashant/DevDocsHub
# fixed Navigation and added aout, languages and contact page

**Title: Fix Navigation Buttons and Update Templates**
Description:
This PR addresses issues with the navigation buttons and updates multiple templates in the application. It ensures that buttons redirect to the correct pages for About, Languages, and Contact. Additionally, I have made updates to the corresponding templates, urls.py, and views.py to enhance overall functionality.

**Issue:**
Fixes #55

**Problem:**
The application lacked the corresponding templates for Languages and Contact, making it impossible for users to navigate effectively.

**Fix:**
Updated the HTML templates to include:
About page
Languages page
Contact page
Ensured that the navigation buttons correctly redirect to their respective pages.
Modified urls.py to include routes for the new pages.
Updated views.py to handle requests for the About, Languages, and Contact pages properly.
Steps to Test the Fix:
Navigate to the home page.
Test all buttons on different screen sizes to ensure responsiveness and functionality.
Check that all new templates load without errors.

https://github.com/user-attachments/assets/76e80720-0cbc-4faf-bdeb-55b7443e310a
